services/payment-service/app/routes/payments.py

```
@@ -0,0 +1,332 @@
+    # random.random()은 [0.0, 1.0) 균등 분포 → chaos_failure_rate 이하면 강제 거절
+    is_chaos_failure = (
+        settings.chaos_failure_rate > 0 and random.random() < settings.chaos_failure_rate
+    )
+
+    process_start = datetime.now(UTC)
+    processed_at = datetime.now(UTC)
+
+    # ⑤ Chaos DB 슬로우쿼리 시뮬레이션
+    if settings.chaos_db_slowquery:
+        # 실제 슬로우쿼리 대신 sleep으로 시뮬레이션
+        await asyncio.sleep(random.uniform(1.0, 3.0))
```

Ruff (0.15.12)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)